app/src/main/java/org/p2p/wallet/solend/data/SolendUserDepositsRemoteRepository.kt

```diff
@@ -0,0 +1,26 @@
+package org.p2p.wallet.solend.data
+
+import org.p2p.wallet.infrastructure.network.provider.TokenKeyProvider
+import org.p2p.wallet.sdk.facade.SolendSdkFacade
+import org.p2p.wallet.sdk.facade.model.SolendCollateralAccountResponse
+import org.p2p.wallet.sdk.facade.model.SolendPool
+import org.p2p.wallet.sdk.facade.model.SolendUserDepositResponse
+import org.p2p.wallet.utils.Base58String
+import org.p2p.wallet.utils.toBase58Instance
+
+class SolendUserDepositsRemoteRepository(
```

Comment:
mb?
```suggestion
class SolendDepositsRemoteRepository
```

Reply:
looks like that we are getting all the deposits in the Solend ecosystem instead of user ones
i can rename but add `user` in the method names

Reply:
Kostya is working with this class, decided not to change it

Reply:
in general this class is out of scope. 
I have made a lot of changes here already